packages/babel-helper-create-class-features-plugin/src/fields.js

```diff
@@ -300,8 +303,47 @@ function buildPrivateInstanceFieldInitSpec(ref, prop, privateNamesMap) {
 }
 
 function buildPrivateStaticFieldInitSpec(prop, privateNamesMap) {
-  const { id } = privateNamesMap.get(prop.node.key.id.name);
+  const privateName = privateNamesMap.get(prop.node.key.id.name);
+  const { id, getId, setId, initAdded } = privateName;
   const value = prop.node.value || prop.scope.buildUndefinedNode();
```

Comment:
I suggest move this line immediately before where `value` is used so we can early return for non-accessor class private methods.

Reply:
Updated.